scripts/dotnet.py

```diff
@@ -284,7 +284,8 @@ def restore(self,
         cmdline = [
             'dotnet', 'restore',
             self.csproj_file,
-            '--packages', packages_path
+            '--packages', packages_path,
+            '/p:UseSharedCompilation=false', '/p:BuildInParallel=false', '/m:1',
```

Comment:
Can we just get a comment as to why we have this line? Just in case we look in 9 months and can't remember why we enfore non-parallel builds.

Reply:
good point, I've added the comments 👍